src/components/HotspotEditorModal/HotspotTextStyleTab/HotspotTextStyleTab.story.jsx

```diff
@@ -66,6 +66,7 @@ export const Default = () => {
         onDelete={(change) => {
           action('onDelete')(change);
         }}
+        translateWithId={jest.fn()}
```

Comment:
Same comment about the jest function in our stories. Is this intentional?

Reply:
thanks for catching